### PR TITLE
Remove scenario modification tools for non-owners

### PR DESCRIPTION
**Pull request by kdeloach**
Hides modifications tools for non-owners and renames a few arguments
to make things easier to understand.

How to test:
1. You should be able to see all modification controls on all owned projects.
2. You should only be able to see the precipitation slider on non-owned projects (or if you are logged out).
3. Changing the precip slider on non-owned projects should update the results panel, but should *not* save anything.

Connects #357
Connects #397

### CODE REVIEW

File: src/mmw/js/src/modeling/views.js [hunk 1]
```diff
@@ -429,20 +430,21 @@ var ScenarioDropDownMenuView = Marionette.CompositeView.extend({
 var ToolbarTabContentView = Marionette.CompositeView.extend({
     model: models.ScenarioModel,
     template: scenarioToolbarTabContentTmpl,
-    collection: models.ModificationsCollection,
+    collection: models.ModelPackageControlsCollection,
```
Comment (kdeloach) [inline src/mmw/js/src/modeling/views.js]: This was actually a mistake before that I just noticed.

File: src/mmw/js/src/modeling/models.js [hunk 2]
```diff
@@ -9,6 +9,14 @@ var Backbone = require('../../shim/backbone'),
 var ModelPackageControlModel = Backbone.Model.extend({
     defaults: {
         name: ''
+    },
+
+    // Return true if this is an input control and false if it is a
+    // modification control.
+    isInputControl: function() {
```
Comment (kdeloach) [inline src/mmw/js/src/modeling/models.js]: This is poorly named and will be address when I split modifications & input controls.

**Comment by kdeloach** [comment]
Updated so input controls are always visible.

**Comment by kdeloach** [comment]
Updated with test instructions.

File: src/mmw/js/src/modeling/views.js [hunk 3]
```diff
@@ -770,6 +779,10 @@ var ResultsTabContentsView = Marionette.CollectionView.extend({
     }
 });
 
+function isEditable(scenario) {
```
Comment (caseypt) [inline src/mmw/js/src/modeling/views.js]: Consider making this an instance method on scenario.

**Comment by caseypt** [comment]
Consider adding some tests, or adding potential tests to the running list here: https://github.com/WikiWatershed/model-my-watershed/issues/393

**Comment by caseypt** [comment]
+1. Tested. I was able to edit and run scenarios on an owned, public project. Then in an incognito window I loaded that project. The modification controls were gone, but the precipitation slider was still there. I was able to change the slider and run models.

**Comment by caseypt** [comment]
Also, the precipitation slider now shows `0.0"` when it is all the way to the left.

File: src/mmw/js/src/modeling/views.js (hunk 3 above)
Comment (kdeloach) [inline src/mmw/js/src/modeling/views.js]: Forgot to address this before merging. I wasn't sure if it made sense to make this an instance method, considering it relies on a global variable. I could rewrite this to something like `isEditableBy(user_id)` and make it an instance method on Scenario?

Comment (caseypt) [inline src/mmw/js/src/modeling/views.js]: +1 to your proposal.

Comment (kdeloach) [inline src/mmw/js/src/modeling/views.js]: Cool, I'll fix this tomorrow.

**Pull request merged**